packer/builder/azure/arm/config.go

```diff
@@ -50,10 +50,9 @@ type Config struct {
 	ResourceGroupName string `mapstructure:"resource_group_name"`
 	StorageAccount    string `mapstructure:"storage_account"`
 
-	// Packer
-	UserName string `mapaccount:"username"`
-
 	// Runtime Values
+	UserName             string
+	Password             string
 	tmpAdminPassword     string
```

Comment:
I put them there so I can resolve what the *real* user name and password are.  Depending on what communicator you're using you'll get source from a different variable (ssh_username vs. winrm_username).  Adding all of that logic elsewhere, and just producing a UserName/Password **seems** more beneficial.